### PR TITLE
Add debianization files

### PR DESCRIPTION
**Pull request by nbuchwitz**
Follow-up of #3

### CODE REVIEW

File: debian/copyright [hunk 1]
```diff
@@ -0,0 +1,37 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+
+Files: *
+Copyright: 2022 Nicolai Buchwitz <n.buchwitz@kunbus.com>
```
Comment (iluminat23) [inline debian/copyright]: Copyright is Kunbus GmbH
You are "just" the Author

File: debian/copyright [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+ .
+ You should have received a copy of the GNU General Public License
+ along with this program. If not, see <https://www.gnu.org/licenses/>
+ .
+ On Debian systems, the complete text of the GNU General
+ Public License version 2 can be found in "/usr/share/common-licenses/GPL-2".
+
+License: LGPL-3+
```
Comment (iluminat23) [inline debian/copyright]: What is now licensed with GPL-2 and what is LGPL-3+? Or is it dual-licensed?

**Review by iluminat23 — CHANGES_REQUESTED** [review]

File: debian/copyright (hunk 1 above)
Comment (RamiGspo) [reply debian/copyright]: I used that with the maya package:

Copyright: KUNBUS GmbH

File: debian/copyright (hunk 2 above)
Comment (nbuchwitz) [reply debian/copyright]: dual-licensed. The idea is to allow customers to integrate the device info data into their applications even if this closed.

Comment (iluminat23) [reply debian/copyright]: Why LGPL-3 not LGPL-2?
Also the License is missing in the source directory.

Comment (nbuchwitz) [reply debian/copyright]: From what I know LGPLv3 allows to include code licensed under Apache or MIT. Also it seems to be more explicit in terms of lawyer-language.

Comment (nbuchwitz) [reply debian/copyright]: @iluminat23 do you have a suggestion to add both licenses to the file? Append the second license? separate files with license name suffix?

Comment (iluminat23) [reply debian/copyright]: As fare a I know (L)GPL v3 requires, that an owner of the device must be enabled to run his changed code. What makes it the least favorite license on all embedded devices.
Hmmm I've never did anything like this. How does the rest of the world these things? If we know that we can ignore the rest of the world and do it in a sane way ;)
If we want to be permissive we could also use MIT. Then nobody has to care about any license incompatibilities.

Comment (nbuchwitz) [reply debian/copyright]: That would be a show-stopper. I'm not a professional licensee, so maybe MIT might be a better idea? I really don't care about anyone using this library for their profit maximization :)

Comment (iluminat23) [reply debian/copyright]: We could do this:
`debian/*` GPL-2
`*` MIT
This would be a real debian like way to do it.

Comment (nbuchwitz) [reply debian/copyright]: ... an license file with MIT?

Comment (iluminat23) [reply debian/copyright]: yes

File: debian/copyright [hunk 3]
```diff
@@ -0,0 +1,26 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+
+Files: debian/*
+Copyright: 2022 KUNBUS GmbH
+License: GPL-2+
```
Comment (iluminat23) [inline debian/copyright]: I think you need at least the short GPL in here. Or we just make the whole thing MIT. It's easier. The debian/* GPL was a not so bright idea.

Comment (nbuchwitz) [reply debian/copyright]: Done

**Review by iluminat23 — APPROVED** [review]

**Pull request merged**